### DISCUSSION TITLE
Displaying the day of the week in the title

### DISCUSSION THREAD

**Discussion by stwebage** [vkurko/calendar#194] [Q&A] [answered]
Good evening, tell me how to display the day of the week in the central part in the day view, in addition to dada: Monday - Tuesday, etc.
![Снимок экрана 2023-10-25 в 18 01 39](https://github.com/vkurko/calendar/assets/66872911/bab6e24b-41ce-4096-90b2-67f5f16cca39)

**Comment by vkurko** [comment, marked as answer]
Please check [titleFormat](https://github.com/vkurko/calendar#titleformat) option.

**Reply by Fosnez** [reply]
Is that confirmed to be working? I've tried all the below (separetly, not at once) and none of them seem to change the displayed date on any of the views? Am i being an idiot?

```
eventCalendar.setOption('dayheaderformat', {weekday: 'long', month: 'numeric', day: 'numeric'});
eventCalendar.setOption('dayheaderformat',  {dateStyle: 'full'});
eventCalendar.setOption('dayheaderformat', function(date){return 'Bar';});
eventCalendar.setOption('titleformat', {dateStyle: 'full'});            
eventCalendar.setOption('titleformat', {weekday: 'short', year: 'numeric', month: 'long', day: 'numeric'});
eventCalendar.setOption('titleformat', {weekday: 'long'});
eventCalendar.setOption('titleformat', function(start, end){return 'Foo';});
```

**Reply by vkurko** [reply]
@Fosnez The problem in your case is that the option names are case sensitive. You should use `titleFormat` instead of `titleformat`, and then everything should work.

**Reply by Fosnez** [reply]
aha! So I am being an idiot... Thankyou @vkurko :)
